src/frontend/web_application/src/components/form/DatePickerGroup/presenter.jsx

```diff
@@ -0,0 +1,69 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import DatePicker from 'react-datepicker'; // https://github.com/Hacker0x01/react-datepicker
+import 'react-datepicker/dist/react-datepicker.css';
+import moment from 'moment';
+import classnames from 'classnames';
+
+import './style.scss';
+
+class DatePickerGroup extends Component {
+  static propTypes = {
+    calendarClassName: PropTypes.string,
+    className: PropTypes.string,
+    id: PropTypes.string.isRequired,
+    inputClassName: PropTypes.string,
+    label: PropTypes.string,
+    locale: PropTypes.string,
+    onDateChange: PropTypes.func.isRequired,
+    selected: PropTypes.instanceOf(Date),
+  };
+  static defaultProps = {
+    calendarClassName: null,
+    className: null,
+    inputClassName: null,
+    label: null,
+    locale: undefined,
+    selected: null,
+  };
+
+  state = {
+    startDate: null,
+  };
+
+  render() {
+    const {
+      calendarClassName,
+      className,
+      id,
+      inputClassName,
+      label,
+      locale,
+      onDateChange,
+      selected,
+      ...props
+    } = this.props;
+
+    const selectedProps = selected ? moment(selected) : false;
```

Comment:
In react-datepicker: `selected: PropTypes.object,`  
and since `typeof null === 'object'` bamm :)

Just for the sport, I believe if a prop is a boolean it is passed through, since it's a valid value.
I would rather set it to `null` or `undefined` to not pass it.